src/renderers/dom/shared/ReactDOMClientInjection.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactDOMClientInjection
+ */
+
+'use strict';
+
+var BeforeInputEventPlugin = require('BeforeInputEventPlugin');
+var ChangeEventPlugin = require('ChangeEventPlugin');
+var DOMEventPluginOrder = require('DOMEventPluginOrder');
+var EnterLeaveEventPlugin = require('EnterLeaveEventPlugin');
+var EventPluginHub = require('EventPluginHub');
+var EventPluginUtils = require('EventPluginUtils');
+var ReactBrowserEventEmitter = require('ReactBrowserEventEmitter');
+var ReactDOMComponentTree = require('ReactDOMComponentTree');
+var ReactDOMEventListener = require('ReactDOMEventListener');
+var SelectEventPlugin = require('SelectEventPlugin');
+var SimpleEventPlugin = require('SimpleEventPlugin');
+
+var alreadyInjected = false;
+
+function inject() {
+  if (alreadyInjected) {
+    // TODO: This is currently true because these injections are shared between
+    // the client and the server package. They should be built independently
+    // and not share any injection state. Then this problem will be solved.
```

Comment:
We can, but we should probably remove it from all such files (there's a few). It's not relevant since flat bundles.